### PR TITLE
Fix-s2-stac

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[STAC-Aggregation liefert Fehler](https://trello.com/c/oj70fFWk/625-stac-aggregation-liefert-fehler)

File: services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs [hunk 1]
```diff
@@ -247,18 +247,37 @@ impl SentinelS2L2aCogsMetaData {
...
+                if start == prev_start {
+                    log::debug!("duplicate start time: {}", start);
+                    start_times[i - 1] + 1
+                } else {
+                    TimeInstance::from(start)
+                }
+            };
+
+            start_times.push(real_start);
+        }
```
Comment (ChristianBeilschmidt) [inline services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs:266]: Please write a test for that. This won't work if you have three consecutive timestamps
`[1, 1, 1]`, then your algorithm would produce `[1, 2, 1]` I guess.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs (hunk 1 above)
Comment (jdroenner) [reply services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs]: [a87638a](https://github.com/geo-engine/geoengine/pull/480/commits/a87638a2c5618301fb26ea4990bfdcf79f790006)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**